cosmic-applet-status-area/src/components/app.rs

```diff
@@ -188,6 +189,10 @@ impl cosmic::Application for App {
                 }
                 Task::none()
             }
+            Msg::RightPressed(id) => {
+                self.menus[&id].ctx_menu_activate();
+                Task::none()
```

Comment:
To be sure I'm not sure about what to return here.
I need to better familiarize myself with tasks (and async Rust in general)